### ISSUE TITLE
Put all the collections as tabs on the bottom (or top)

### ISSUE THREAD

**Issue by rmlockwood** [cdfarrow/flextools#25]
In FLExTrans we are always switching back and forth between collections. It would be so convenient if all the collections were listed as tabs say at the bottom of the window and then one could just click on say the Tools tab to more quickly have the Tools modules in view. Instead of 3 clicks, it's just one click. Most of the time the user doesn't need to see all the various modules and be able to rearrange them. At times he or she might want to modify a collection, so that capability should still be there. It wouldn't be that often, though, so we may not need 'Collections' as a big button. It could be a menu choice.

**Comment by rmlockwood** [comment]
We just got done with a FLExTrans workshop and it reminded me every day that we would really appreciate this enhancement. New users were constantly confused about how to get to another collection to run a particular module. Usually they would forget it is a double-click. Sometimes, after they opened the collections manager window, they would see the module they want on the upper-right pane and try to execute it by double-clicking on it. It was painful to watch!

Even if the solution isn't tabs, just some way to quickly and easily get from one collection to another would be much appreciated.

**Comment by bbryson** [comment]
For me, it would be nice if we could "open" or "close" modules, and then when they are open, they show up in tabs.  I don't necessarily want all the collections open at once.  For FLExTrans, often I just want "All Steps" and "Tools" open at the same time.  Sometimes I might also want the "Testbed" collection.

If that's too complicated, even just allowing the Tools collection to be a tab that we could always get to, no matter which other collection we have open, would help.

Or having all collections showing up as tabs would also work.  But I agree--we need a better approach for when there are at least two collections that we are switching back and forth between quite frequently.

**Comment by cdfarrow** [comment]
> For me, it would be nice if we could "open" or "close" collections.
Yes, that is a good idea. If it showed all collections as tabs then it could be too much if you have a lot of collections.

**Comment by cdfarrow** [comment]
I've added tabs for collections, which can be opened by selecting them in the Collections Manager, and closed with a right-click menu. The tabs appear in alphabetical order along the top (below the toolbar).
I've also added a new ini file option "HideCollectionsButton = True" as per the request for the Collections button to be removed from the toolbar for FlexTrans users. This defaults to False, so the button is visible for regular users.
